src/interfaces.ts

```diff
@@ -66,6 +66,7 @@ export interface FormResults {
   status?: number;
   type?: string;
   link?: string;
+  name?: string;
```

Comment:
Why is name optional?

Reply:
I'm not sure of the best place to put the full name yet, either from the server or from the app. So optional for now.